DualView/app/build.gradle

```diff
@@ -0,0 +1,52 @@
+/*
+ *
+ *  * Copyright (c) Microsoft Corporation. All rights reserved.
+ *  * Licensed under the MIT License.
+ *
+ */
+
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+
+android {
+    compileSdkVersion 30
```

Comment:
not sure if in other modules these values are placed also in a `dependencies.gradle` file. Please, consider it.